packages/docs-site/src/content/docs/docs/plugins/bitwarden.mdx

```diff
@@ -0,0 +1,144 @@
+
+```typescript title='apps/some-service/.dmno/config.mts'
+import { BitwardenSecretsManagerDmnoPlugin } from '@dmno/bitwarden-plugin';
+
+// 💉 inject the already initialized plugin instead of re-initializing it
+const bitwardenPlugin = BitwardenSecretsManagerDmnoPlugin.injectInstance('bitwarden');
+```
+
+
+------------
+
+## Setup Project & Secrets
+
+If you are already using Bitwarden Secrets Manager, you likely aleady have existing [projects](https://bitwarden.com/help/projects/) that contain [secrets](https://bitwarden.com/help/secrets/). If so, now would be a good time to review how they are all organized. If not, you should create at least one project, as each secret can have a parent project it belongs to, and access can be granted to projects rather than managing each secret individually.
+
+:::tip[Use projects to segment access]
+You should use multiple projects to segment your secrets following the Principle of Least Privilege. This means at the very least, you should have one project for ultra-sensitive production secrets, and another for everything else. How much more you want to break things up is up to you and will depend on your security requirements.
+
+It is technically possible to create secrets without using projects and/or to assign permissions at the individual secret level, but we do not recommend it.
+:::
+
+## Setup Machine Account & Access Tokens
+
+Machine accounts can be granted access to projects, and each machine account can have multiple access tokens with optional expiration. How you want to manage this is up to you, but a sensible approach could be:
+
+- Production machine account has access to all projects
+  - single access token is used at a time
+- Staging/CI machine account has access to all projects except ultra-sensitive prod secrets
+  - each environment/CI/external tool could have a unique access token
+- Dev machine account has access to secrets needed for local dev
+  - each developer could have a unique access token
+
+Expiring tokens are obviously more secure, but require overhead of rolling those tokens, which must be done manually. Not wanting to cause an outage, you may want to roll them manually without the pressure of a potentially forgotten expiry date. To roll without downtime, create a new token, redeploy, and then decommission the previous token.
+
+### 
+
+
+Machine account access token(s) now serve as your "secret-zero" - which grants access to the rest of your sensitive config stored in Bitwarden. It must be set locally and in deployed environments, but it is sensitive so we must pass in the value as an _override_ rather than storing it within the config. Locally this usually means storing it in your [`.env.local` file](/docs/guides/env-files/) and on a deployed environment you'll usually set it within some kind of UI, wherever you would normally pass in environment variables.
```

Comment:
```suggestion
Machine account access tokens now serve as your _secret-zero_ - which grants access to the rest of your sensitive config stored in Bitwarden. It must be set locally and in deployed environments, but it is sensitive so we must pass in the value as an _override_ rather than storing it within the config. Locally, this usually means storing it in your [`.env.local` file](/docs/guides/env-files/) and on a deployed environment you'll usually set it within some kind of UI, wherever you would normally pass in environment variables.
```